tests/test_user_management.py

```diff
@@ -1,19 +1,25 @@
 import json
```

Comment:
As another comment I would try to name the tests for the files `./pylana/<name>.py` `./tests/test_<name>.py/` for the user management unless there's a good reason not to. This makes life a bit easier. I already failed to do this with the semantics creation, so maybe we can change that name too with `git mv ./test/test_create_semantics.py ./tests/test_utils.py` (this way git will recognise the files as the same and keep the version history from before the renaming).